libr/bin/obj.c

```diff
@@ -57,9 +58,10 @@ R_API RBinObject *r_bin_object_new(RBinFile *binfile, RBinPlugin *plugin, ut64 b
 		if (sz < bsz) {
 			bsz = sz;
 		}
-		o->bin_obj = plugin->load_bytes (binfile, bytes + offset, sz,
+		success = plugin->load_bytes (binfile, &o->bin_obj, bytes + offset, sz,
```

Comment:
no need to have a variable that is only used once you can just` if (!plugin->load_bytes( ...)`

Reply:
I'm going to remove it, rebase and send the updated PR.